core/pom.xml

```diff
@@ -16,6 +16,12 @@
     <name>rest-utils</name>
 
     <dependencies>
+        <dependency>
+            <groupId>org.eclipse.jetty.websocket</groupId>
+            <artifactId>javax-websocket-server-impl</artifactId>
+            <version>9.2.22.v20170606</version>
```

Comment:
can we put this version into dependency management in the parent pom to make sure we keep it in sync with the main jetty version?